code/game/gamemodes/miniantags/abduction/abductee_objectives.dm

```diff
@@ -131,9 +131,6 @@
 /datum/objective/abductee/party
 	explanation_text = "You're throwing a huge rager. Make it as awesome as possible so the whole crew comes... OR ELSE!"
 
-/datum/objective/abductee/pets
```

Comment:
Can you readd this as `/datum/objective/abductee/cool_pets`?

Reply:
sure